src/main/java/ee/carlrobert/codegpt/settings/service/llama/LlamaSettings.java

```diff
@@ -26,6 +28,17 @@ public LlamaSettingsState getState() {
   @Override
   public void loadState(@NotNull LlamaSettingsState state) {
     this.state = state;
+    // Catch if model's name has changed which could lead to
+    // HuggingFaceModel or PromptTemplates not being found
+    if (this.state.getHuggingFaceModel() == null) {
```

Comment:
What kind of compatibility issues does it solve? I'm not quite sure I'm following.

Reply:
Right! Thank you for explaining :)